<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QwcNewsWidget</class>
 <widget class="QWidget" name="QwcNewsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>News</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="pageWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="fNews">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="overwriteMode">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="fBtnRefresh">
           <property name="text">
            <string>Refresh</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/icons/16x16/view-refresh.png</normaloff>:/icons/16x16/view-refresh.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="fNewsStatus">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">0 articles, 0 bytes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="fBtnDelete">
           <property name="toolTip">
            <string>Delete all news articles.</string>
           </property>
           <property name="text">
            <string>Purge</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/icons/16x16/edit-delete.png</normaloff>:/icons/16x16/edit-delete.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fBtnPost">
           <property name="toolTip">
            <string>Post a new article.</string>
           </property>
           <property name="text">
            <string>Post...</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/icons/16x16/mail-message-new.png</normaloff>:/icons/16x16/mail-message-new.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>160</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="fProc">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="loadingLabel">
         <property name="text">
          <string>Loading news...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Compose a new article and press &quot;Post&quot; to send it to the server.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="composeMessage">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_2">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnComposeCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/icons/16x16/process-stop.png</normaloff>:/icons/16x16/process-stop.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnComposePost">
           <property name="text">
            <string>Post</string>
           </property>
           <property name="icon">
            <iconset resource="application.qrc">
             <normaloff>:/icons/16x16/mail-message-new.png</normaloff>:/icons/16x16/mail-message-new.png</iconset>
           </property>
           <property name="shortcut">
            <string>Enter</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
